tools/crm_resource.c

```diff
@@ -2031,15 +2037,34 @@ main(int argc, char **argv)
                 params = pcmk__strkey_table(free, free);
                 get_meta_attributes(params, rsc, current, data_set);
 
+            } else if (pcmk__str_eq(options.attr_set_type, ATTR_SET_ELEMENT, pcmk__str_casei)) {
+
+                for (xmlAttrPtr a = pcmk__xe_first_attr(rsc->xml); a != NULL; a = a->next) {
+                    const char *p_name = (const char *) a->name;
+                    const char *p_value = ((a == NULL) || (a->children == NULL)) ? NULL :
+                                        (const char *) a->children->content;
+                    if (strcmp(p_name, options.prop_name) == 0) {
+                        if (p_value != NULL) {
+                            out->begin_list(out, NULL, NULL, "Attributes");
+                            out->list_item(out, options.prop_name, "%s", p_value);
+                            out->end_list(out);
+                        } else {
+                            out->err(out, "Attribute '%s' not found for '%s'", p_name, rsc->id);
+                        }
+                    }
+                }
+
             } else {
                 params = pcmk__strkey_table(free, free);
                 pe__unpack_dataset_nvpairs(rsc->xml, XML_TAG_UTILIZATION, NULL, params,
                                            NULL, FALSE, data_set);
             }
 
-            rc = out->message(out, "attribute-list", rsc, options.prop_name, params);
-            if (free_params) {
-                g_hash_table_destroy(params);
+            if (params) {
```

Comment:
For clarity about each variable's data type, we avoid implicit booleans in conditions in new code. We like to explicitly compare against NULL (`if (params != NULL)`), or against 0 if the variable is an integer.

That guideline is here: https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Development/singlehtml/#control-statements-if-else-while-for-switch